cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -2,13 +2,16 @@
 
 set -e
 
-source ${KUBEVIRTCI_PATH}/cluster/images.sh
-
 if [ "${KUBEVIRTCI_RUNTIME}" = "podman" ]; then
     _cli="pack8s"
 else
-    _cli_container="${KUBEVIRTCI_GOCLI_CONTAINER:-kubevirtci/${IMAGES[gocli]}}"
-    _cli="docker run --privileged --net=host --rm ${USE_TTY} -v /var/run/docker.sock:/var/run/docker.sock ${_cli_container}"
+    _cli=./cli
+    # If this is not kubevirtci release tarball build gocli and use, it will
+    # Use :latest providers
```

Comment:
Would you assume to have a pipeline, where a pre-step creates all clusters?

Reply:
That's it only the pipeline would have permission for `:latest` so latest are safe.